### DISCUSSION TITLE
Having problems with struct payment method

### DISCUSSION THREAD

**Discussion by ryantorsparks** [Alkaar/resy-booking-bot#115] [Q&A]
Thanks for posting this code firstly! I'm just having one small problem with the payment method for the final booking step, I keep getting
```
'{"status": 400, "code": null, "message": "Invalid data received.", "data": {"struct_payment_method": "invalid"}}'
```
my struct_payment_method looks like this:
```
{"id": 1234567}
```

was wondering if anyone has any ideas on what I might be doing wrong there? thanks

**Comment by Alkaar** [comment]
Double check your credit card is not expired? Someone mentioned having this issue at some point.
